packages/mock/src/MockStore.ts

```diff
@@ -420,7 +420,8 @@ export class MockStore implements IMockStore {
     if (value !== undefined) return value;
 
     const type = this.getType(typeName);
-    const interfaces = type.getInterfaces();
+    // GraphQL 14 Compability
```

Comment:
Are you sure interfaces was not part of GQL 14? Moreover `getInterfaces` was [there](https://github.com/graphql/graphql-js/blob/v14.0.0/src/type/definition.js#L668) at least since 14.0.0 🤔

Reply:
Not on interface fields, interfaces can implement interfaces tho in v15

Reply:
Ah yes yes! Indeed thanks